pkg/reco/workflow.go

```diff
@@ -168,3 +170,15 @@ func pickSafestPolicy(p1, p2 *Policy) *Policy {
 		return p2
 	}
 }
+
+func transformTargetRecoConfig(targetRecoConfig *v1alpha1.HPAConfiguration) *v1alpha1.HPAConfiguration {
+	if targetRecoConfig == nil {
+		return nil
+	}
+	maxReplicas := targetRecoConfig.Max
+	minReplicas := targetRecoConfig.Min
+	if maxReplicas >= 3 && minReplicas < 3 {
```

Comment:
can we make this config driven instead of hardcoding?

Reply:
ok, naming it minRequiredReplicas in the config?

Reply:
sure